File: market/src/main/java/com/wanted/market/member/repository/MemberRepository.java
package com.wanted.market.member.repository;

import com.wanted.market.member.domain.Member;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface MemberRepository extends JpaRepository<Member, Integer> {
    Boolean existsByEmail(String email);

    Member findByEmail(String email);
}

File: api/src/main/kotlin/wanted/market/api/domain/member/repository/MemberRepository.kt
package wanted.market.api.domain.member.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import wanted.market.api.domain.member.entity.Member

@Repository
interface MemberRepository : JpaRepository<Member, Long> {
    fun findByEmailAndPassword(email: String, password: String): Member?
    fun findByEmail(email: String): Member?
}
